### PR TITLE
{lib}[foss/2022a] CPC2 v0.1 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by deniskristak**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by deniskristak** [comment]
one of dependencies for https://github.com/vscentrum/vsc-software-stack/issues/106

File: easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    ('Python', '3.10.4'),
+    ('LIBSVM', '3.30'),
+    ('Biopython', '1.79'),
+]
+
+install_cmd = 'cp -a CPC2*/{bin,data} %(installdir)s/'
+
+sanity_check_paths = {
+    'files': ['bin/CPC2.py'],
+    'dirs': [],
```
Comment (boegel) [inline easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb:35]: also check for `data` dir?

```suggestion
    'dirs': ['data'],

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — DISMISSED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3144.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/acb6f615d59e01d20ca613dc745e4918 for a full test report.

File: easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
+# Updated: Denis Kristak (Inuits)
+# Update to a Python3 version
+easyblock = 'PackedBinary'
+
+name = 'CPC2'
+version = '0.1'
+local_commit = '813487e'
+
+homepage = 'http://cpc2.cbi.pku.edu.cn'
```
Comment (Micket) [inline easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb:10]:
```suggestion
homepage = 'http://cpc2.gao-lab.org/'
```
site claims to have migrated away from there, redirected here

File: easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+description = """ a fast and accurate coding potential calculator based on sequence intrinsic features  """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+# original sources at http://cpc2.cbi.pku.edu.cn/download.php are currently not available
+# download from github mirror
+source_urls = ['https://github.com/lucventurini/CPC2/archive/']
```
Comment (Micket) [inline easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb:17]: This seems to be available? https://github.com/gao-lab/CPC2_standalone/releases
I was directed here by going to the old website, which redirected to the new gao-lab website, which redirected here. 
This seems much much newer as well compared to the version 0.1 in the lucventurini repo, and this one has been ported to python 3 already.

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb (hunk 3 above)
Comment (boegel) [reply easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb:17]: @deniskristak @smoors Thoughts on going with a more recent version available via https://github.com/gao-lab/CPC2_standalone/releases ?

Comment (deniskristak) [reply easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb:17]: @boegel just pushed an updated version using the newer source. I had to change version too, so I'm removing the original files.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/51b1e5c1dea6603423446b90806c93c8 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Looks okay to me now. 
Agreed @Micket ?

File: easybuild/easyconfigs/c/CPC2/CPC2-1.0.1-foss-2022a.eb [hunk 4]
```diff
@@ -0,0 +1,39 @@
+easyblock = 'PackedBinary'
+
+name = 'CPC2'
+version = '1.0.1'
+
+homepage = 'http://cpc2.cbi.pku.edu.cn'
```
Comment (Micket) [inline easybuild/easyconfigs/c/CPC2/CPC2-1.0.1-foss-2022a.eb:6]:
```suggestion
homepage = 'http://cpc2.gao-lab.org/'
```

this wasn't fixed?

File: easybuild/easyconfigs/c/CPC2/CPC2-1.0.1-foss-2022a.eb [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+description = """ a fast and accurate coding potential calculator based on sequence intrinsic features  """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+# original sources at http://cpc2.cbi.pku.edu.cn/download.php are currently not available
+# download from github mirror
```
Comment (Micket) [inline easybuild/easyconfigs/c/CPC2/CPC2-1.0.1-foss-2022a.eb:12]: It's not the mirror anymore, it's the official release page now
```suggestion
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/09582f73c530fd6a234e28410a396999 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
Going in, thanks @deniskristak!

**Pull request merged**